### PR TITLE
[enhancement] Improve curl response message #50

### LINKED ISSUE #50: [refactor] Improve curl response mesage
This change was originally requested here:
https://github.com/openwisp/openwrt-openwisp-monitoring/pull/42#pullrequestreview-708960736

>Logged response can be improved
Here's an example output: Data not sent successfully. Response code is {"detail":"Authentication credentials were not provided."}403.. The message is readable but it's weird. The response body is {"detail":"Authentication credentials were not provided."} while the response status code is 403. Can you turn this into something like:
The response received was: 403 {"detail":"Authentication credentials were not provided."}?

### PR DESCRIPTION
**Pull request by devkapilbansal**
Closes #50

### CODE REVIEW

**Comment by devkapilbansal** [comment]
https://github.com/openwisp/openwrt-openwisp-monitoring/pull/34#pullrequestreview-733493335

**Comment by devkapilbansal** [comment]
OpenWISP Package Compiling

**Comment by devkapilbansal** [comment]
OpenWISP Package Compiling

File: openwrt-openwisp-monitoring/files/monitoring.agent [hunk 1]
```diff
@@ -84,7 +84,7 @@ set_url_and_curl(){
 
   URL="$BASE_URL/api/v1/monitoring/device/$UUID/?key=$KEY"
 
-  CURL_COMMAND="curl -s -w "%{http_code}""
+  CURL_COMMAND="curl -s -w "%{http_code}" --output "$TMP_DIR"/response.txt"
```
Comment (nemesisdesign) [inline openwrt-openwisp-monitoring/files/monitoring.agent:87]: I think the repetition of `"$TMP_DIR"/response.txt"` should be avoided

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

File: openwrt-openwisp-monitoring/files/monitoring.agent (hunk 1 above)
Comment (devkapilbansal) [reply openwrt-openwisp-monitoring/files/monitoring.agent:87]: :grimacing: Removed

**Review by nemesisdesign — APPROVED** [review]

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**